bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -387,15 +405,26 @@ for tnum in template_ids:
             for ifo in trigs.ifos:
                 addtime = times[ifo][ids[ifo]]
                 addtriggerid = tids[ifo][ids[ifo]]
-                data['%s/time' % ifo] += [addtime]
-                data['%s/trigger_id' % ifo] += [addtriggerid]
-            data['stat'] += [cstat]
-            data['decimation_factor'] += [dec]
-            data['timeslide_id'] += [slide]
-            data['template_id'] += [numpy.repeat(tnum, len(slide))]
-
+                local_data['%s/time' % ifo] += [addtime]
+                local_data['%s/trigger_id' % ifo] += [addtriggerid]
+            local_data['stat'] += [cstat]
+            local_data['decimation_factor'] += [dec]
+            local_data['timeslide_id'] += [slide]
+            local_data['template_id'] += [numpy.repeat(tnum, len(slide))]
             start1 += args.batch_singles
         start0 += args.batch_singles
+    return local_data
+
+if args.nprocesses == 1:
+    ldatas = list(map(process_template, list(template_ids)))
+else:
+    p = multiprocessing.Pool(args.nprocesses)
+    ldatas = p.map(process_template, list(template_ids))
+
+logging.info('merging data from the templates')
+for ldata in ldatas:
+    for key in data:
+        data[key] += ldata[key]
```

Comment:
This looks right, but I guess here the proof is in testing. Can you comment on how the pool stuff was tested?

Reply:
Just ran by hand to verify the output. This is somewhat different than I ran in production (that was more manual, hence this patch so I don't have to do that again).